frontend/src/metabase/components/OnClickOutsideWrapper.jsx

```diff
@@ -19,10 +19,22 @@ export default class OnClickOutsideWrapper extends ClickOutComponent {
             // NOTE: this actually doesn't seem to be working correctly for popovers since PopoverBody creates a stacking context
             ReactDOM.findDOMNode(this).parentNode.style.zIndex = popoverStack.length + 2; // HACK: add 2 to ensure it's in front of main and nav elements
         }, 10);
+
+        // we unfortunately need to have this function here to access the proper context
+        const handleKeyPress = (event) => {
+            const escKey = 27; // the char code for the esc key
```

Comment:
I'd probably just get rid of the `const` and inline it, keeping the comment. Or make it a top level constant `const ESC_KEY = 27;`